client/src/client.rs

```diff
@@ -1122,6 +1122,31 @@ pub mod transaction {
     }
 }
 
+pub mod role {
+    //! Module with queries for roles
+    use super::*;
+
+    /// Get query to retrieve all roles
+    pub const fn all() -> FindAllRoles {
+        FindAllRoles::new()
+    }
+
+    /// Get query to retrieve all role ids
+    pub const fn all_ids() -> FindAllRoleIds {
+        FindAllRoleIds::new()
+    }
+
+    /// Get query to retrieve role by id
```

Comment:
```suggestion
    /// Get query to retrieve a role by its id
```

Reply:
fixed, thank you for your suggested changes.